### DISCUSSION TITLE
Sponsor fee from open source projects

### DISCUSSION THREAD

**Discussion by josesimoes** [community/community#52038] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hi,

Just read your annoucement about the incoming changes in sponsors. 
Most of them seem very positive and congrats on making it easier and smooth the process of sponsoring open source projects.

Now my concern: the fees that will be charged to sponsoring organizations. I'm with the core team of [.NET nanoFramework](https://github.com/nanoframework) we are an open source project accepting donations from GitHub sponsors and Opencollective.
We are sponsoring another open source project that we depend on. With this new fee we'll be "overcharged". I totally get the credit card processing fee, but the processing fee seems a bit too much... In particular for situation like the one I'm describing: an Open Source project sponsoring another Open Source project. If it was an enterprise or a company, I (kind of) get it. 
For the sake of the discussion: I still think these 3% processing fee it's a bit too much.

**Comment by anjuan** [comment]
Thanks for your feedback! We’re always looking to our maintainers for suggestion and we’ve added this onto our backlog to look into.
